views/index.html

```diff
@@ -9,85 +9,83 @@
   </head>
   <body>
 
-    <div class="calculator">
-      <div class="display">
-        <p id="numbers">0</p>
-    </div>
+    <div class="page-container">
+      <div class="calculator-body">
+      <div class="display"><p id="numbers">0</p></div>
+        <div id="calculator-numbers">
+          <div class="calclulator-row1">
+            <div class="calculator-button">
+              <button class="calculator-ac">AC</button>
+            </div>
+            <div class="calculator-button">
+              <button class="calculator-positive-negative">+/-</button>
+            </div>
+            <div class="calculator-button">
+              <button class="percent">%</button>
+            </div>
```

Comment:
You don't need to wrap your buttons in a div like this.  You can just use the buttons directly, or just use a div.